src/ClientInterface.php

```diff
@@ -47,10 +47,11 @@ public function captureLastError(?Scope $scope = null): ?EventId;
     /**
      * Captures a new event using the provided data.
      *
-     * @param array<string, mixed>|Event $payload The data of the event being captured
-     * @param Scope|null                 $scope   An optional scope keeping the state
+     * @param Event          $event The event being captured
+     * @param EventHint|null $hint  May contain additional information about the original exception
```

Comment:
The description is not accurate, the fact that this object may contain the original exception is just a detail, we should describe what the `$hint` is and not what it contains